src/client/styles/components/pie-hero.scss

```diff
@@ -0,0 +1,28 @@
+//
```

Comment:
Please move this file to a separate PR. It seems it's not related to webpack migration in any way, and this Pr is already > 2000 lines of code. It's also missing tests.

Reply:
Oops, this file isn't supposed to be here, bad split. Sorry!